### PR TITLE
Add new resource form page

### PR DESCRIPTION
**Pull request by sarayourfriend**
Adds a new resource form page.

To test, go to the deploy preview, click the link to visit the new resource page and submit the form with test data. Verify either in the email inbox or in Netlify that the form has been picked up.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <meta charset="utf-8">
 <meta http-equiv="x-ua-compatible" content="ie=edge">
-<title>Not 911 - {{ page.title }}</title>
+<title>{{ page.title }} | Not 911</title>
```
Comment (vertexactual) [inline _includes/head.html:3]: Huh, why this change?

File: new-resource.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <p>
+        <label class="required">Resource name: <input type="text" required name="name" /></label>
+    </p>
+    <p>
+        <!-- Don't use type="url" here because it's too strict and requires a schema which some users might not bother to enter and could be confusing -->
+        <label>Resource URL (if available): <input type="text" name="url" /></label>
+    </p>
+    <p>
+        <label class="required" for="phone">Resource phone:</label>
+        <aside>If there are additional phone numbers, please list them in the dscription</aside>
```
Comment (vertexactual) [inline new-resource.md]: "If there are additional phone numbers, please list them in the description"

File: new-resource.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <input id="phone" type="tel" required name="phone" />
+    </p>
+    <p>
+        <label for="resource-email">Resource email:</label>
+        <aside>If available, any contact you have at the organization behind the resource or a general purpose email for the organization</aside>
+        <input id="resource-email" type="email" name="resource-email" />
+    </p>
+    <p>
+        <label class="required" for="category">Resource category:</label>
+        <aside>Please choose at least one. To select multiple on PC, hold control (or command if on a mac) and click on each option you wish to select.</aside>
```
Comment (vertexactual) [inline new-resource.md]: "Please choose at least one. To select multiple, hold Control (on PC), or Command (On Mac), and click on each option you wish to select."

**Review by vertexactual — APPROVED** [review]
All looks good. Unable to commit atm, but commented a couple verbiage changes.

File: _includes/head.html (hunk 1 above)
Comment (sarayourfriend) [reply _includes/head.html:3]: Browser tabs only have so much space and putting the site name first before the page title makes the page title get cut off. The favicon already gives the context that this is the not 911 tab so there's no need to put the site title first.

Most websites put the page title first too, so I think it was just us not thinking closely about this when it was first implemented.

File: new-resource.md (hunk 3 above)
Comment (sarayourfriend) [reply new-resource.md]: Done in https://github.com/OrcaCollective/not911.me/pull/52/commits/0def96f7f212b6e3c6312226e029a9f1b7478854

File: new-resource.md (hunk 2 above)
Comment (sarayourfriend) [reply new-resource.md]: Done in https://github.com/OrcaCollective/not911.me/pull/52/commits/0def96f7f212b6e3c6312226e029a9f1b7478854

File: new-resource.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+title: New Resource
+stylesheet: /new-resource.css
+---
+
+<a href="/">Back to the list</a>
+
+<h2>New Resource Request</h2>
+
+<p>
+    Thank you for taking an interest in helping us expand this list of resources. To request a new resource be added, please fill the form below. Please note that the information sent in the form is not private and can be read by our web host, Netlify. There's nothing we can do about this at this time without significantly increasing the cost to run this website. If you'd prefer to contact us directly, please email us at <a href="mailto:techblocsea@protonmail.com">techblocsea@protonmail.com</a>.
```
Comment (AetherUnbound) [inline new-resource.md:12]: Love that you included the note about netlify!

**Review by AetherUnbound — COMMENTED** [review]
This looks awesome, thank you!

When I submit the form, there ends up being a huge white section on the bottom of the site in dark mode. Is that a quick CSS fix?

image

**Comment by sarayourfriend** [comment]
@AetherUnbound Good catch, I hadn't tested in dark mode. It was a one line fix :+1:
